R/factorize.R

```diff
@@ -22,15 +22,15 @@ factorize_secuTrial <- function(x, ...) UseMethod("factorize_secuTrial", x)
 #'
 #' @return \code{secuTrialdata} object with extra variables in forms for factors (names are appended with \code{.factor})
 #' @export
-#'
-#' @examples
 factorize_secuTrial.secuTrialdata <- function(object) {
   if (!object$export_options$refvals_separate) {
     ifelse(options()$stringsAsFactors,
            saf <- "\nCategorical variables are probably factors already (options()$stringsAsFactors == TRUE)\n",
            saf <- "\nCategorical variables are probably strings (options()$stringsAsFactors == FALSE)\n")
     stop(saf, "Recommend saving reference values to seperate table in export")
   }
+  if (object$export_options$factorized) warning("already factorized - any changes will be lost")
```

Comment:
@aghaynes basically the same question as for the dated label. Any reason to rerun if it has already been done?

Reply:
As above, I wanted to avoid errors. I don't see any reason why someone would want to run it twice for factors though...